RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronIDProducer.cc

```diff
@@ -212,16 +124,19 @@ void LowPtGsfElectronIDProducer::produce(edm::StreamID, edm::Event& event, const
     edm::ValueMap<float>::Filler filler(*ptr);
     filler.insert(gsfElectrons, output[iname].begin(), output[iname].end());
     filler.fill();
-    reco::GsfElectronRef ele(gsfElectrons, 0);
+    //reco::GsfElectronRef ele(gsfElectrons, 0);
```

Comment:
remove commented-out code.

Reply:
Many thanks for the comments @jpata, all have been acted on, there's just one query above. 

Regarding this:
> If possible, can you also do a before/after timing check with `--customise Validation/Performance/TimeMemoryInfo.py`? Thanks!

Does it not make more sense to wait until we have the final model ready? Essentially all the extra timing will be due to its evaluation, which will differ (perhaps substantially) from the current "placeholder" model). 

@slava77 Yes, I can sequence the module using the bParking modifier, but I'm not exactly sure of the best place to add the necessary python snippets. I can give it a go.